ui/src/ListCall.tsx

```diff
@@ -0,0 +1,32 @@
+import React, {FC} from 'react';
+import {List, ListItem, ListItemProps, ListItemText} from "@material-ui/core";
+import {Call} from "./Call";
+
+function ListItemLink(props: ListItemProps<'a', { button?: true }>) {
+
+    return <ListItem button divider component="a" {...props}/>;
+}
+
+interface ListCallProps {
+    calls: Array<Call>
+    currentCall:Call
+    setCurrentCall: React.Dispatch<React.SetStateAction<Call>>
+}
+
+export const ListCall:FC<ListCallProps> = ({calls, currentCall, setCurrentCall}) => {
+    const handleClick  = (c:Call) => {
+            setCurrentCall(c)
+    }
+
+    return (
+        <List component="nav" aria-label="secondary mailbox folders">
+            {calls.map((v) => {
```

Comment:
A thing that I've noticed comparing to golang, is that single char var are not really a recommendation in JS or generally I never worked with that approach in JS. I totally don't get why is that a thing in golang to be honest :P 

```suggestion
            {calls.map((call) => {
```